### ISSUE TITLE
MD011 flagging on MkDocs `table-reader` plugin

### ISSUE THREAD

**Issue by kattni** [rvben/rumdl#96]
Hello! 

I'm relatively certain the answer here is to manually disable this rule on the lines in question. I wanted to bring it up here anyway. Feel free to agree with me there and close this as resolved.

`table-reader` enables reading from a CSV file, and generating a Markdown table through another plugin called `macros` that allows specific Jinja syntax in MkDocs Markdown. The call to `table-reader` looks like this:

```{{ pd_read_csv("../data/widgets_by_platform.csv", na_filter=False, usecols=[4,5,6,7,8,9,10])[pd_read_csv("../data/widgets_by_platform.csv")[["ComponentName"]].isin(["Application"]).all(axis=1)] | convert_to_md_table }}```

For obvious reasons, `MD011` is flagging this as reverse link syntax. 

I wanted to mention it in case you had an idea for a way to make `rumdl` compatible with `table-reader`.

Otherwise, I'll do exactly what I suggested to begin with and disable the rule inline.

**Comment by rvben** [comment]
Jinja2 support is now added. Thanks for the issue!
